jsx/Form.js

```diff
@@ -348,26 +351,42 @@ var TextboxElement = React.createClass({
       id: null,
       disabled: false,
       required: false,
+      hasError: false,
+      errorMessage: 'The field is required!',
       onUserInput: function() {
         console.warn('onUserInput() callback is not set');
-      }
+      },
+      onUserBlur: function() {
+        console.warn('onUserBlur() callback is not set');
```

Comment:
i would not make this the default for this function since there are generally few instances of needing to rely on blur events. i suggest using an empty do-nothing function instead

Reply:
Will do!